light_curves/code_src/panstarrs.py

```diff
@@ -249,6 +249,11 @@ def panstarrs_get_lightcurves(coords_list, labels_list, radius):
     """
         
     df_lc = MultiIndexDFObject()
+    
+    #first convert sample_table to coords_list, labels_list
+    coords_list = [(row['objectid'] - 1, row['coord']) for row in sample_table]
```

Comment:
Same as above; should be:

```suggestion
    coords_list = [(row['objectid'], row['coord']) for row in sample_table]
```